setup.cfg

```diff
@@ -0,0 +1,5 @@
+[wheel]
+universal = 1
```

Comment:
(for my future reference: `https://wheel.readthedocs.io/en/stable/index.html?highlight=universal#defining-the-python-version` )